packages/react-scripts/scripts/build.js

```diff
@@ -184,16 +184,16 @@ function build(previousFileSizes) {
       }
       const build = path.relative(process.cwd(), paths.appBuild);
       console.log(`The ${chalk.cyan(build)} folder is ready to be deployed.`);
-      console.log('You may also serve it locally with a static server:');
+      console.log('You may serve it with a static server:');
       console.log();
       if (useYarn) {
-        console.log(`  ${chalk.cyan('yarn')} global add pushstate-server`);
+        console.log(`  ${chalk.cyan('yarn')} global add serve`);
       } else {
-        console.log(`  ${chalk.cyan('npm')} install -g pushstate-server`);
+        console.log(`  ${chalk.cyan('npm')} install -g serve`);
       }
-      console.log(`  ${chalk.cyan('pushstate-server')} build`);
+      console.log(`  ${chalk.cyan('serve')} -s build`);
       console.log(
-        `  ${chalk.cyan(openCommand)} http://localhost:${process.env.PORT || 9000}`
+        `  ${chalk.cyan(openCommand)} http://localhost:${process.env.PORT || 3000}`
```

Comment:
@gaearon The great thing about `serve` is that it cares about the port already being in use. If it is, it will show a big red message and use an open one. The URL will be copied to the clipboard automatically in addition.

As the last great thing, it will also show the URL on the network.

Here's how it looks by default:

![screen shot 2017-03-08 at 15 40 50](https://cloud.githubusercontent.com/assets/6170607/23708274/af75da6c-0415-11e7-9b6d-c1288ee29341.png)

And that's how it looks when the default port is already in use AND no `--port` is specified:

![screen shot 2017-03-08 at 15 41 09](https://cloud.githubusercontent.com/assets/6170607/23708293/c0e3587e-0415-11e7-864c-50068d839166.png)